source/common/dmtbinfo2.c

```diff
@@ -678,6 +678,77 @@ ACPI_DMTABLE_INFO           AcpiDmTableInfoLpit0[] =
     ACPI_DMT_TERMINATOR
 };
 
+/* 17: core interrupt controller */
+ACPI_DMTABLE_INFO           AcpiDmTableInfoMadt18[] =
```

Comment:
The names of these are all off-by-one in the integer. Also these belong in the MADT Subtables section below (that already exists and has 0 through 16), not on the end of the LPIT Subtables.

Reply:
Yes, you're right, these code are definitely wrong putting here, and I'll move it to madt.

Thanks.